test/colony-network-mining.js

```diff
@@ -1933,7 +1933,7 @@ contract("ColonyNetworkMining", accounts => {
 
       badClient = new MaliciousReputationMinerExtraRep(
         { loader: contractLoader, minerAddress: OTHER_ACCOUNT, realProviderPort: REAL_PROVIDER_PORT, useJsTree },
-        27,
+        32,
```

Comment:
32 is the correct update number for the origin reputation. 27 is for the colony-wide child skill
```
updateNumber 26
key 0xdce090e1cbe4aaa0d2be9e5a77c5a3a6f43d79e800000000000000000000000000000000000000000000000000000000000000029df24e73f40b2a911eb254a8825103723e13209c
amount -5000000000016
updateNumber 27
key 0xdce090e1cbe4aaa0d2be9e5a77c5a3a6f43d79e800000000000000000000000000000000000000000000000000000000000000050000000000000000000000000000000000000000
amount -5000000000016
updateNumber 28
key 0xdce090e1cbe4aaa0d2be9e5a77c5a3a6f43d79e800000000000000000000000000000000000000000000000000000000000000010000000000000000000000000000000000000000
amount -5000000000016
updateNumber 29
key 0xdce090e1cbe4aaa0d2be9e5a77c5a3a6f43d79e800000000000000000000000000000000000000000000000000000000000000040000000000000000000000000000000000000000
amount -5000000000016
updateNumber 30
key 0xdce090e1cbe4aaa0d2be9e5a77c5a3a6f43d79e800000000000000000000000000000000000000000000000000000000000000059df24e73f40b2a911eb254a8825103723e13209c
amount -5000000000016
updateNumber 31
key 0xdce090e1cbe4aaa0d2be9e5a77c5a3a6f43d79e800000000000000000000000000000000000000000000000000000000000000019df24e73f40b2a911eb254a8825103723e13209c
amount -5000000000016
updateNumber 32
key 0xdce090e1cbe4aaa0d2be9e5a77c5a3a6f43d79e800000000000000000000000000000000000000000000000000000000000000049df24e73f40b2a911eb254a8825103723e13209c
amount -4931280523281
```